### ISSUE TITLE
disabled selinux broke installation

### ISSUE THREAD

**Issue by guillaume-philippon** [quattor/aii#157]
Hi,

Since 15.12.0 version , when you put ```/system/aii/osinstall/ks/selinux```to ```disabled``` aii-ks add -selinux* to package to install. 

That broke kickstart post install with ```-s option not exist``` error.

I don't know what we want to do with this option but it's not correctly done.

```
# cat quattor/aii/ks/config.pan
"/system/aii/osinstall/ks/packages" = {
    if (value('/system/aii/osinstall/ks/selinux') == 'disabled') {
        append('-selinux*');
    };
    SELF;
};
```

**Comment by stdweird** [comment]
what OS is this?

**Comment by jouvin** [comment]
This happened in Cent0S 7.2. Conversely to what @guillaume-philippon said, there was no change in 15.2 compared to 15.4 (and probably 15.8). The change is that this is the first 7.2 machine we installed from scratch and it seems. The reason is not really clear as `yum` still accept a package starting with a `-` as long as the end of options (before the first package) is marked with `--` (but it seems it has always been like this).
The easy workaround is to define AII_OSINSTALL_SELINUX='permissive' in the site config. I'm not sure there is any drawback to have it permissive rather than disabled during installation and that can be changed when configuring the machine...

**Comment by stdweird** [comment]
@guillaume-philippon @jouvin can you send me the kickstart file that was used? we set this and have no issues with installations.

**Comment by stdweird** [comment]
our relvant part of the kickstart
```
...
%packages --ignoremissing
@Base
@Core
NetworkManager
bind-utils
curl
initscripts
lsof
nc
openssh
openssh-server
perl-AppConfig
perl-CDB_File
perl-Crypt-SSLeay
perl-DBI
perl-GSSAPI
perl-IO-String
perl-Pod-POM
perl-Template-Toolkit
perl-URI
perl-XML-Parser
perl-libwww-perl
perl-parent
wget
yum-plugin-priorities
yum-plugin-versionlock
-selinux*

%end

...
```
(there is an extra newline between the `-selinux*` and the `%end`).
and `/system/aii/osinstall/ks/end_script` is set to `%end` in the profile.

**Comment by stdweird** [comment]
@guillaume-philippon ping?

**Comment by stdweird** [comment]
@guillaume-philippon should be fixed by #171 and in 16.4. Please reopen this issue if this is not the case
